src/exampleTypes/CheckableTermsStatsTable.js

```diff
@@ -0,0 +1,49 @@
+import _ from 'lodash/fp'
+import F from 'futil-js'
+import React from 'react'
+import { observer } from 'mobx-react'
+import { Column } from '../layout/ExpandableTable'
+import TermsStatsTable from './TermsStatsTable'
+
+let CheckableTermsStatsTable = observer(
+  ({
+    criteria,
```

Comment:
by destructuring this up here, it means `criteria` isn't in `props` and so isn't passed to the underlying `TermsStatsTable`